### ISSUE TITLE
Imposible to add multiple recipients

### ISSUE THREAD

**Issue by lifebarier** [thunderbird/thunderbird-android#5996]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

5.807

### Where did you get the app from?

Google Play

### Android version

11

### Steps to reproduce

1. Start creating new email
2. Add long recipient
3. Click message text field
4. Click to field again
5. Add one more long recipient do not add space at the end (note adding two recipients without step 3 is less consistent but reproducible)
6. Click on message field again
7. Click on to field

### Expected behavior

Same as after step 3 - I can add new recipient

### Actual behavior

Email details of last recipient shows up ( <Unknown Recipient> etc...), and keyboard hides. It is impossible to add new email. Clicking on text field and back to 'to' field yelds same buggy behavior
Side note - Why 'x' at top right of this popup deletes recipient? 'x' there means close not 'delete'
Side note two - I can't find how to edit recipient email one it is shown as 'bubble'

**Comment by numericOverflow** [comment]
I've got the same issue. Can't paste a list of email addresses into the TO/CC/BCC fields. 

Pops up an error that can't parse addresses.

**Comment by cketti** [comment]
@numericOverflow: This is a different issue. See #2961

**Comment by cketti** [comment]
I can't reproduce this. Can you provide a screen recording?

**Comment by lifebarier** [comment]
> I can't reproduce this. Can you provide a screen recording?

This should explain it. But it does not happen 100% of time. took me 3 tries.

https://user-images.githubusercontent.com/1889485/166163102-3319296a-348d-4d05-b314-149691e7b4e3.mp4

**Comment by Tc-001** [comment]
I believe you are hitting the `X` button here:
![image](https://user-images.githubusercontent.com/55956895/167315397-9fdb6a1c-1bec-4c59-b28b-68aa9f9cc98c.png)

Clicking that `X` will remove the recipient. 

It may be useful to change it to a trashcan icon to make it's purpose clearer?

**Comment by lifebarier** [comment]
>I believe you are hitting the X button here:

Yes, I hit that to illustrate that when I want to add extra contact it appears preventing addition of any more contacts and obvious 'x' (aka 'close') is deleting, and not closing (as I was hoping it would close and let me add more contacts). Changing icon to trash icon would make it clearer (still bad. add trash icon and leave 'x' which closes this popup) **but it does not solve main problem**.

Main issue is that **it is not possible to add more recipients** as shown in my video and described in description because that thing containing 'x' appears when I focus 'To' field and prevents editing.
Did you even read bug description?

**Comment by Tc-001** [comment]
I see what you mean, sorry for the previous message!
I can also reproduce this, where the popup closes the keyboard.

**Comment by mhf1998** [comment]
Hello everyone. This bug recently happened to me as well, and I was able to figure out the scenario.
This bug is quite annoying...
But, I managed to find a clear scenario and its cause:

When you type the email address of a recipient and then confirm it (either by pressing Next or by tapping the message input field), that recipient gets added to the recipient list in a separate box. After the recipient is added in its own box, an extra space is automatically inserted right after it.
and If, after writing the message body, we go back to the “To” field, we see a single space between the previous recipient and the cursor where we’re typing.
So far everything works fine.

But… if we delete that space (for any reason), the bug starts. Deleting that space causes a problem: when we try to switch between other fields and the “To” field and then add a new recipient in the “To” field, we can’t. This bug prevents the keyboard from appearing for that “To” field — every time we move from another field into “To,” the keyboard no longer pops up. Instead, the recipient information dialog for the previous recipient appears .

Did I make the scenario clear?
To summarize: when you type a recipient and confirm it (by pressing Next or by moving between fields) so that the recipient becomes a confirmed recipient (the oval/pill element), the app automatically inserts a space after it. If that space is deleted, the app no longer allows the keyboard to appear or the user to type a new recipient.

Were you able to follow the scenario? @wmontwe @kewisch 
Please investigate this bug — it’s been present for years and breaks a core workflow.
 Thanks.

**Comment by mhf1998** [comment]
Hi. Since this issue has already been addressed and tracked again in #10367, which explicitly references #5996 as a related issue and overlaps with the same behavior, I believe this issue can also be closed.
